App/components/listItems/NotificationListItem.tsx

```diff
@@ -15,21 +15,12 @@ interface Props {
 const NotificationListItem: React.FC<Props> = ({ notification }) => {
   const navigation = useNavigation()
 
-  const decideTitle = () => {
-    switch(notification.type) {
-      case 'CredentialRecord':
-        return 'Credential Offer'
-      default:
-        return ''
-    }
-  }
-
   return (
     <TouchableOpacity
       style={styles.container}
       onPress={() => navigation.navigate('Credential Offer', { notification })}
     >
-      <Text style={styles.title}>{decideTitle()}</Text>
+      <Text style={styles.title}>{notification.type || 'Notification'}</Text>
       <Icon name="chevron-right" color={textColor} size={30} />
     </TouchableOpacity>
   )
```

Comment:
This would likely need subsequent updates to account for different notification types, right?

Reply:
na, just whatever type is on the object from the system will be rendered, This will change with the cred flow updates though

Reply:
Yes, and when you tap on the notification right now it's set to take you to the credential offer screen, even it's not an offer I think

Reply:
oh! correct, we'll have to make a more generic notificationDetails screen or something like that